docs/Plugins-evals.md

```diff
@@ -0,0 +1,241 @@
+Plugin: evals
+==========
+
+## APT3 DIY ATT&CK Evaluation Round 1
+
+ATT&CK Evaluations use an Operational Flow to describe the logical ordering to which procedures take place during the evaluation. Round 1's Operational Flow is found [here](https://attackEval.mitre.org/methodology/round1/operational-flow.html).
```

Comment:
This link is broken and I'm not sure what the updated link should be.

Reply:
https://attackevals.mitre.org/methodology/round1/operational-flow.html